packages/runtime/runtime-definitions/api-report/runtime-definitions.api.md

```diff
@@ -397,11 +397,14 @@ export interface ISummaryTreeWithStats {
 
 // @public
 export interface ITelemetryContext {
-    get(prefix: string, property: string): TelemetryEventPropertyType;
-    push(prefix: string, property: string, value: TelemetryEventPropertyType[]): void;
+    get(prefix: string, property: string): TelemetryEventPropertyTypeExt;
+    push(prefix: string, property: string, value: string | number | boolean | undefined | {
+        [key: string]: // Flat objects can have the same properties as the event itself
+        string | number | boolean | undefined | (string | number | boolean)[];
+    }): void;
     serialize(): string;
-    set(prefix: string, property: string, value: TelemetryEventPropertyType): void;
-    setMultiple(prefix: string, property: string, values: Record<string, TelemetryEventPropertyType>): void;
+    set(prefix: string, property: string, value: TelemetryEventPropertyTypeExt): void;
```

Comment:
We want to avoid exposing the `Ext` ones on any public API if possible.

...Shoot I see some already are public.